### ISSUE TITLE
Function arguments

### ISSUE THREAD

**Issue by ArneBachmann** [evhub/coconut#392]
```Python
def tryOrDefault(func:() -> Any, default:Any) -> Any:
  try: return func()
  except: return default


def a(): raise Exception()

print(tryOrDefault(() -> True, False))
print(tryOrDefault(a, True))
```

```
ash@mega:~/Desktop/svn/projects/sos> coconut-develop test.coco --mypy
Compiling         test.coco ...
Compiled to       test.py .
/usr/lib/python3.4/site-packages/coconut/stubs/__coconut__.pyi:76: error: Cannot find module named 'backports.functools_lru_cache'
/usr/lib/python3.4/site-packages/coconut/stubs/__coconut__.pyi:76: note: (Perhaps setting MYPYPATH or using the "--ignore-missing-imports" flag would help)
/usr/lib/python3.4/site-packages/coconut/stubs/__coconut__.pyi:77: error: Module has no attribute "lru_cache"
Coconut: Exiting due to MyPy error.
ash@mega:~/Desktop/svn/projects/sos> coconut-develop test.coco --mypy --ignore-missing-imports
Left unchanged    test.py (pass --force to override).
/usr/lib/python3.4/site-packages/coconut/stubs/__coconut__.pyi:77: error: Module has no attribute "lru_cache"
Coconut: Exiting due to MyPy error.
```

**Comment by eindiran** [comment]
I'm having trouble reproducing this. I copied the code in your example into a file, and tried to build it, and got the following complaint:

`test.py:19: error: Name 'Any' is not defined`

So I added the line: `from typing import Any`

Upon compiling with `coconut-develop test.coco --mypy` I get success; I can run the result, which prints:
`True
True`

I notice that it's python3.4 in paths there. This might be a silly suggestion, but have you tried running mypy without coconut involved using 3.4? I would give it a try but I only have 2.7 and 3.6 installed.
